### ISSUE TITLE
[REQUEST] using pycord instead of discord.py

### ISSUE THREAD

**Issue by 5cat** [interactions-py/interactions.py#371]
**What is the feature you're proposing?**
Since discord.py is not maintained and features such as threads are not in the latest stable release of discord.py it might be a good idea to discord-interactions support pycord as a backend instead of discord.py since it is just a maintained fork of discord.py.

what do you think?

**Comment by Toricane** [comment]
Please join the [server](https://discord.gg/interactions) and look in #news

TL;DR: we are going independent

**Comment by Catalyst4222** [comment]
~~Since v4 is focusing on being an independent library, and v3 is effectively static now, there's no reason to change the supported library~~ got ninja'd 

nextcord already supports v3, and [there's a fix](https://github.com/Devoxin/Lavalink.py/pull/113/files) that will let v3 work with any fork

**Comment by 5cat** [comment]
> Please join the [server](https://discord.gg/interactions) and look in #news
> 
> TL;DR: we are going independent

Thank you for the news! that is very assuring. the fast response and the low number of open issues + open pull requests is very amazing.



> ~Since v4 is focusing on being an independent library, and v3 is effectively static now, there's no reason to change the supported library~ got ninja'd
> 
> nextcord already supports v3, and [there's a fix](https://github.com/Devoxin/Lavalink.py/pull/113/files) that will let v3 work with any fork

Thank you for letting me know

**Comment by i0bs** [comment]
On a more rather unofficial note -- BobDotCom and I may be looking into some plans to create bridging compatibility and support between our two libraries in the future, although nothing is set in stone.
